### PR TITLE
[Release] 9.0.0-alpha.1

### PR DESCRIPTION
**Pull request by sunshinejr**
Hey guys! 🎉 

I'm really excited to say that we are almost there! Just need to double check if this merge got everything we need for 9.0.0. This is a really long one, but don't be scared! Most of that codebase was reviewed in multiple PRs so this is just a final check to see if it's alright. Even if its not, we still got alpha/beta tests so no worries.

Cheers!

**Edit:** Also, related: #1075

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
 
 - Enabled the "Allow app extension API only" flag.
 - Added an optional `requestDataFormatter`in `NetworkLoggerPlugin` to allow the client to interact with the request data before logging it.
+- **Breaking Change**: Added support to get the response (if any) from `MoyaError`.
```
Comment (pedrovereza) [inline Changelog.md]: We normally list all Breaking Changes first

File: Sources/Moya/TargetType.swift [hunk 2]
```diff
@@ -27,6 +27,9 @@ public protocol TargetType {
 
     /// Whether or not to perform Alamofire validation. Defaults to `false`.
     var validate: Bool { get }
+
+    // The headers to used in the request.
```
Comment (pedrovereza) [inline Sources/Moya/TargetType.swift]: Just noticed I left a typo here 😄
We should use either `The headers used` or `The headers to be used`

**Comment by sunshinejr** [comment]
Good ones, @pedrovereza! Updated both `Changelog` & comment for `headers` in `TargetType` 👍

**Comment by sunshinejr** [comment]
Okay, I also fixed most of the issues from SwiftLint. The rest is either from Demo (which we might update as well, in another PR) or from things we know exist but we need time to refactor (like long `MoyaProviderSpec`)

**Comment by sunshinejr** [comment]
Hey @Moya/contributors, just a reminder that I'd love it if we could get another pair of eyes on that one. Pushing an alpha release this week would be 💯

**Comment by sunshinejr** [comment]
Thank you, @SD10! @pedrovereza, do you feel like you'd have time to validate this one as well? :)

**Comment by sunshinejr** [comment]
Thank you so much, y'all! 🎉

**Pull request merged**